znai-cli/src/main/java/com/twosigma/znai/cli/ZnaiCliApp.java

```diff
@@ -180,7 +180,12 @@ private PageModifiedTimeStrategy pageModifiedTimeStrategy() {
 
     private void createNew() {
         ConsoleOutputs.out(Color.BLUE, "scaffolding new documentation");
-        DocScaffolding scaffolding = new DocScaffolding(Paths.get("znai"));
+
+        Path pathToScaffold = config.isSourceRootSet() ?
```

Comment:
Can we move this logic into config.getSourceRoot(), then you don't have to make `isSourceRootSet()`?

    Path pathToScafoold = config.getSourceRoot();

    getSourceRoot() {
         return isSourceRootSet ? getSourceRoot() : Paths.get("znai");
    }

Reply:
Not really. Source root has a value set by default. Most of the time we need that value. But for “new” command we need to check if the value was ever set by a user.